### PR TITLE
Bug 1916271: openstack known issues: ControlPlane anti-affinity

### PR DESCRIPTION
**Pull request by pierreprinetti**
Add a note on the limitations of the soft-anti-affinity server group
that the installer creates.

/label platform/openstack

### CODE REVIEW

File: docs/user/openstack/known-issues.md [hunk 1]
```diff
@@ -81,3 +81,18 @@ The teardown playbooks provided for UPI installation will not delete:
...
+A long-standing OpenStack Compute issue prevents the "soft-anti-affinity"
+policy to be correctly applied when instances are created in parallel. The
+Installer places the initial Control plane nodes in a Server group with
+"soft-anti-affinity" policy and creates the first three sequentially to work
+around the problem. Additional Control plane nodes beyond the first three are
+created, according to the "replicas" property, in a later phase. These nodes
+are not guaranteed to be created sequentially.
+
+When anti-affinity is a requirement, it is advised to only install with three
+nodes, and then add any further Control plane nodes by assignign them a new
```
Comment (mandre) [inline docs/user/openstack/known-issues.md:96]:
```suggestion
control plane nodes, and then add any further control plane nodes by assigning them a new
```

File: docs/user/openstack/known-issues.md [hunk 2]
```diff
@@ -81,3 +81,18 @@ The teardown playbooks provided for UPI installation will not delete:
...
+Server group with "anti-affinity" policy. Use the `ServerGroupID` property of
```
Comment (mandre) [inline docs/user/openstack/known-issues.md:97]: Does it mean the Nova issue affects the "soft-anti-affinity" but not the "anti-affinity" policy? If so, perhaps we should be more explicit.

Comment (pierreprinetti) [reply docs/user/openstack/known-issues.md]: Is this new version any better?

File: docs/user/openstack/known-issues.md (hunk 1 above)
Comment (pierreprinetti) [reply docs/user/openstack/known-issues.md]: OK, de-capitalized "Control plane" and fixed the typo

File: docs/user/openstack/known-issues.md (hunk 2 above)
Comment (mandre) [reply docs/user/openstack/known-issues.md]: It is, thanks.

**Review by mandre — COMMENTED** [review]
/approve

Not `/lgtm`ing right away to give @mdbooth a chance to review.

**Review by mdbooth — APPROVED** [review]
LGTM. Nice and clear.

**Pull request merged**